src/guides/v2.3/frontend-dev-guide/layouts/layout-create.md

```diff
@@ -17,14 +17,24 @@ If the new page has a `3-columns-double-footer` layout, create a custom page-lay
     <update handle="3columns"/>
     <referenceContainer name="page.wrapper">
         <container name="footer-bottom" as="footer-bottom" after="footer" label="Footer Bottom" htmlTag="footer" htmlClass="page-footer-bottom">
-            <container name="footer-bottom-content" as="footer-bottom-content" htmlTag="div" htmlClass="footer content">
-                <block class="Magento\Framework\View\Element\Template" name="report.bugs.bottom" template="Magento_Theme::html/bugreport.phtml"/>
-            </container>
+            <container name="footer-bottom-content" as="footer-bottom-content" htmlTag="div" htmlClass="footer content" />
         </container>
     </referenceContainer>
 </layout>
 ```
 
+Add a block to the container.
+Create the layout `app/design/frontend/<VendorName>/<ThemeName>/Magento_Theme/layout/3-columns-double-footer.xml`.
```

Comment:
Is this a single step?

```suggestion
To add a block to the container, create the layout:

`app/design/frontend/<VendorName>/<ThemeName>/Magento_Theme/layout/3-columns-double-footer.xml`
```

Reply:
@hguthrie thanks for your recommendations, I have fixed it.